app/controllers/insights_cloud/hits_controller.rb

```diff
@@ -0,0 +1,17 @@
+module InsightsCloud
+  class HitsController < ::ApplicationController
+    def index
+      host = Host.where(id: hits_params).first
```

Comment:
```suggestion
      host = Host.find(host_id_param).first
```

`find` will throw a meaningful exception here